pkg/network/vmicontroller/status.go

```diff
@@ -51,6 +51,16 @@ func UpdateStatus(clusterConfig clusterConfigChecker, vmi *v1.VirtualMachineInst
 
 	interfaceStatuses = append(interfaceStatuses, secondaryIfaceStatuses...)
 
+	// virt-handler adds status for interfaces only existing inside the guest that were discovered by QEMU GuestAgent
+	guestInternalIfaceStatuses := vmispec.FilterInterfaceStatuses(
+		vmi.Status.Interfaces,
+		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
+			return ifaceStatus.Name == "" && ifaceStatus.InfoSource == vmispec.InfoSourceGuestAgent
```

Comment:
just curious, why should the name be empty?

Reply:
virt-handler implicitly sets the name of guest-internal interface statuses to an empty string:
https://github.com/kubevirt/kubevirt/blob/7fade41f9138d787476b2e429c86222a2fc951cd/pkg/network/setup/netstat.go#L293-L295